tests/TitleTest.php

```diff
@@ -1119,6 +1119,17 @@ public function testCast_tv_one_episode_one_year()
         $this->assertCount(0, $castMember['role_other']);
     }
 
+    public function testStars_Cast()
+    {
+        $imdb = $this->getImdb('0075314');
+        $stars = $imdb->actor_stars();
+        $castMember = array_find_item($stars, 'imdb', '0000134');
+
+        $this->assertEquals('0000134', $castMember['imdb']);
+        $this->assertEquals('Robert De Niro', $castMember['name']);
+        $this->assertCount(4, $stars);
```

Comment:
I propose changing this to the matrix to save downloading another imdb page just for this test
```suggestion
        $imdb = $this->getImdb();
        $stars = $imdb->actor_stars();
        $castMember = array_find_item($stars, 'imdb', '0000206');

        $this->assertEquals('0000206', $castMember['imdb']);
        $this->assertEquals('Keanu Reeves', $castMember['name']);
        $this->assertCount(4, $stars);
```